app/Http/Controllers/GameController.php

```diff
@@ -231,18 +233,22 @@ public function destroy(Game $game)
 
     public function updateThievesScore(Game $game, int $score)
     {
-        $game->thieves_score = $score;
+        $game->thieves_score += $score;
         $game->save();
 
-        return $game;
+        event(new ScoreUpdatedEvent($game->id, $game->police_score, $game->thieves_score ));
+
+        return $game->thieves_score;
     }
 
     public function updatePoliceScore(Game $game, int $score)
     {
-        $game->police_score = $score;
+        $game->police_score += $score;
```

Comment:
Volgens mij is niet niet gebruikelijk om in een patch een increment uit te voeren in plaats van een set (wijziging).